app/models/request/sample_compound_aliquot_transfer.rb

```diff
@@ -108,13 +106,28 @@ def _create_compound_sample(study, component_samples)
   end
 
   # Default study that the new compound sample will use
+  # Uses the one from the request if it's present,
+  # otherwise, the one from the source aliquots if it's consistent.
   def _default_compound_study
-    _studies.first
+    _initial_study ||
+      begin
+        raise MULTIPLE_STUDIES_ERROR_MSG if _studies.count > 1
+
+        _studies.first
+      end
+  end
+
+  def _initial_study
```

Comment:
Why not just used `initial_study`? This will prevent us from eager loading studies, and force us to look it up again each time.